universal-login-commons/src/core/models/EmailConfirmation.ts

```diff
@@ -0,0 +1,7 @@
+export interface EmailConfirmation {
+  email: string;
+  ensName: string;
+  code: string;
+  created_at?: Date;
+  isConfirmed?: boolean;
```

Comment:
Why is it optional?

Reply:
We have default values at database, so when we want to add a new email to db, then we don't have to pass them

Reply:
Done